### ISSUE TITLE
NVDA screen reader issues reading text on Tools menu

### ISSUE THREAD

**Issue by tmiw** [drowe67/freedv-gui#120]
Per email from Neil Sablatzky on the digitalvoice list, I tested FreeDV 1.5.3 with [NVDA](https://www.nvaccess.org/) and noticed some issues with it being unable to read the contents of the Tools menu. All other menus and application windows seemed to behave correctly at first glance. I think adding tool tips to a few of the Tools menu options will resolve those, but more testing would be good to make sure there aren't any issues I missed.

**Comment by tmiw** [comment]
Actually, it looks like NVDA butchers the Tools menu if I have the mouse cursor to the left of the menu options under it, but reads them correctly if I use the keyboard (or position the mouse cursor to the right of each of the menu options). If anyone else can confirm this behavior, that would be good. 

In any case, at minimum we probably want to ensure each of the menu options are keyboard accessible somehow.

**Comment by tmiw** [comment]
We can probably close this for now considering #121 is now merged into master.
